.github/workflows/update-rasa-pro-image-tag-version.yml

```diff
@@ -0,0 +1,116 @@
+name: update-rasa-pro-image-tag-version.yml
```

Comment:
It might be helpful to update the workflow name to something more descriptive, like “Update Rasa Pro Image Tag,” instead of the current name which includes .yml and is a bit redundant.

Also, adding a short description at the top of the workflow file explaining what the workflow does would be great for future reference and for anyone new to the project.

Reply:
Addressed: https://github.com/RasaHQ/developer-edition-docker-compose/pull/2/commits/8e71a3d709df568dcbec2b00545f96e793224ccc